### PR TITLE
feat(relay): add relay status subcommand

### LINKED ISSUE #221: feat: add aya relay status subcommand
From E2E audit (#169), decision D.

The relay skill uses a raw Python fallback to read profile.json directly for relay status. There's no CLI command for relay-specific health. `aya status` is workspace health, not relay health.

Add `aya relay status` alongside existing `relay list/add/remove`. Output:
- Instance label
- Trusted peers (labels)
- Pending inbox count
- Relay URLs
- Last successful poll timestamp (if available)

Support `--format json` for structured output consistent with other commands.

**Files:** `src/aya/cli.py` (relay subapp)

### PR DESCRIPTION
**Pull request by shawnoster**
## Summary
- Add `aya relay status` command showing instance label, trusted peers, relay URLs, and last poll timestamps
- Supports `--format json` for structured output, `--as` for identity selection
- Closes the CLI gap identified in the E2E audit — the relay skill no longer needs a Python fallback

Closes #221

## Changes
- `src/aya/cli.py`: New `relay_status` command in the `relay_app` subgroup
- `tests/test_cli.py`: 3 tests — text output, JSON shape, empty peers/no poll edge case

## Test plan
- [x] All 701 tests pass
- [x] Lint clean
- [x] Text mode shows instance, peers, relays, last poll (or "(never)")
- [x] JSON mode returns structured dict with expected keys

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: src/aya/cli.py [hunk 1]
```diff
@@ -3065,6 +3065,53 @@ def relay_remove(
...
+    format_: OutputFormat = typer.Option(
+        OutputFormat.AUTO, "--format", "-f", help="Output format: auto (default), text, or json"
+    ),
+) -> None:
+    """Relay health check: identity, trusted peers, relays, last poll."""
+    format_ = resolve_format(format_)
+    p = _load_profile_for_relay(profile)
+
+    # Instance label
+    instance_label = as_ if as_ != "default" else next(iter(p.instances.keys()), "default")
```
[the remark this answers is not in the document]
Comment (shawnoster) [reply src/aya/cli.py]: Fixed — now calls `_resolve_instance(p, as_)` for consistent validation/error handling (41ccd8c).

File: src/aya/cli.py [hunk 2]
```diff
@@ -3065,6 +3065,53 @@ def relay_remove(
...
+
+    if format_ == OutputFormat.JSON:
+        _output_json(
+            {
+                "instance": instance_label,
+                "trusted_peers": trusted_peers,
+                "relays": relays,
+                "last_checked": last_checked,
+            }
+        )
```
[the remark this answers is not in the document]
Comment (shawnoster) [reply src/aya/cli.py:3104]: Intentional scope reduction: live inbox count requires async relay polling which doesn't fit the sync profile-reading pattern of relay subcommands. The relay skill can still get live counts via `aya inbox`. If we want this later, it can be a follow-up.

File: tests/test_cli.py [hunk 3]
```diff
@@ -4170,6 +4170,59 @@ def test_remove_last_requires_force(self, profile_with_instance: Path) -> None:
...
+
+        result = runner.invoke(
+            app,
+            ["relay", "status", "--profile", str(profile_with_instance), "--format", "text"],
+        )
+        assert result.exit_code == 0, result.output
+        assert "(none)" in result.output
+        assert "(never)" in result.output
+
+
```
[the remark this answers is not in the document]
Comment (shawnoster) [reply tests/test_cli.py:4251]: Fixed — added `test_status_with_named_instance` and `test_status_with_unknown_instance_errors` (41ccd8c).
